modules/imgproc/src/smooth.cpp

```diff
@@ -2272,7 +2319,26 @@ static bool ipp_GaussianBlur(InputArray _src, OutputArray _dst, Size ksize,
         if(!ippBorder.m_borderType)
             return false;
 
-        CV_INSTRUMENT_FUN_IPP(::ipp::iwiFilterGaussian, &iwSrc, &iwDst, ksize.width, (float)sigma1, ippBorder);
+        // IW 2017u2 has bug which doesn't allow use of partial inMem with tiling
+        if((((ippBorder.m_borderFlags)&ippBorderInMem) && ((ippBorder.m_borderFlags)&ippBorderInMem) != ippBorderInMem)) {
+            return false;
```

Comment:
I agree with 1. Actually I just copied that logic from ipp_bilateralFilter.

I am not sure about 2 as there is no guarantee that it will be fixed in the next version. Has anyone reported the bug to Intel yet? It seems to me that 3 is the way to go for now.